clinlp/entity.py

```diff
@@ -213,3 +214,41 @@ def __call__(self, doc: Doc):
         doc.set_ents(entities=ents)
 
         return doc
+
+
+def create_term(row, col_term):
+    """Creates a Term object with available attributes."""
+    term = Term("")
+    for attr, value in row.items():
+        if attr == col_term:
+            term.phrase = value
+        if value == value:
+            if value == "True":
+                value = True
+            elif value == "False":
+                value = False
+            elif isinstance(value, float):
+                value = int(value)
+            setattr(term, attr, value)
+    return term
+
+def concept_dict_creator(data, col_label, col_term):
```

Comment:
Het makkelijkste is om functies bijvoorbeeld zo te noemen: `create_concept_dict` (beginnend met een werkwoord), dan is het duidelijk wat ze precies doen